app/views/results/_non_university.html.erb

```diff
@@ -2,7 +2,7 @@
 <% nearest_address = @results_view.nearest_address(course) %>
 <% nearest_location_name = @results_view.nearest_location_name(course) %>
 
-<dt class="govuk-list--description__label">
+<dt class="app-list--description__label">
```

Comment:
As above, not sure this is intended as a modifier of `govuk-list` or a custom app component.